### PR TITLE
Add UID to Logs

### PR DESCRIPTION
**Pull request by dylanuys**
- Miner and Validators display uid in "miner/validator running" log statement
- Fixing a bug in bittensor logging around enabling different log levels 
  - https://github.com/opentensor/bittensor/issues/1836

### CODE REVIEW

**Comment by benliang99** [comment]
Looks good to me, thanks for cleaning up the logging.

**Review by benliang99 — APPROVED** [review]

File: neurons/miner.py [hunk 1]
```diff
@@ -41,6 +35,7 @@ class Miner(BaseMinerNeuron):
 
     def __init__(self, config=None):
         super(Miner, self).__init__(config=config)
+        self.model = None
```
Comment (kenobijon) [inline neurons/miner.py:38]: why is this necessary?

**Review by kenobijon — APPROVED** [review]
LGTM could you just address the question I had about initializing model as none

File: neurons/miner.py (hunk 1 above)
Comment (dylanuys) [reply neurons/miner.py:38]: PEP8, class attributes should be instantiated in __init__

Comment (dylanuys) [reply neurons/miner.py:38]: i.e. make pycharm stop complaining
